src/modules/digits/converters/DigitsConverterType.ts

```diff
@@ -0,0 +1,3 @@
+type DigitsConverter<I = string, O = string> = (value: I) => O;
+
+export default DigitsConverter;
```

Comment:
Please prevent using `export default` and just use the `export`statement, because we should import it with the same name as you exported.